components/funder/src/mutual_credit/outgoing.rs

```diff
@@ -4,7 +4,7 @@ use common::safe_arithmetic::SafeSignedArithmetic;
 
 use proto::funder::messages::{
     CancelSendFundsOp, CollectSendFundsOp, FriendTcOp, RequestSendFundsOp, RequestsStatus,
-    ResponseSendFundsOp, TransactionStage,
+    ResponseSendFundsOp, TransactionStage, is_route_part_valid,
```

Comment:
I thought about something. We might be able to add something like `is_route_part_valid` as a method, which operates on the internal `FriendsRoute.public_keys` using the generic functions. This way we will not need the extra imports of `is_route_part_valid` and `is_route_valid` in various places. I'm not fully sure about this though, what is your opinion?

Reply:
I was against this as I thought both generic and non-generic versions would have to be exported. But they don't have to, so I'll try this now.